rjfoley/style.css

```diff
@@ -0,0 +1,23 @@
+/* Import fonts */
+
+/* Re-styling Basics i.e.: body h1 h2 h3 ... ; */
+
+/* Snippet Title */
+
+.only-text {
+	font-size:24px;
+	line-height: 1.2em;
+	font-family: "Helvetica Neue";
```

Comment:
I would add some default font fall backs.

font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;

Reply:
Aha! Glad you picked up on that. 😄